tools/esp32/Config.mk

```diff
@@ -22,12 +22,11 @@
 # and assemble source files and to insert the resulting object files into an
 # archive.  These replace the default definitions at tools/Config.mk
 
-ifdef BLOBDIR
-	BOOTLOADER=${BLOBDIR}/esp32core/bootloader.bin
-	PARTITION_TABLE=${BLOBDIR}/esp32core/partition-table.bin
-else
-	BOOTLOADER=$(IDF_PATH)/hello_world/build/bootloader/bootloader.bin
-	PARTITION_TABLE=$(IDF_PATH)/hello_world/build/partition_table/partition-table.bin
+ifdef ESPTOOL_BINDIR
+	BOOTLOADER=$(ESPTOOL_BINDIR)/bootloader.bin
+	PARTITION_TABLE=$(ESPTOOL_BINDIR)/partition-table.bin
+	FLASH_BL=0x1000 $(BOOTLOADER)
+	FLASH_PT=0x8000 $(PARTITION_TABLE)
```

Comment:
I don't know if concatenating those values in a variable really helps...
I would prefer keeping them separated and, in the future, turning the base address a new config to support different families in this same script.

Reply:
@Ouss4 I agree with @gustavonihei it is better to create: FLASH_BL_OFFSET=0x1000 and FLASH_PT_OFFSET=0x8000 and add it on final command.

Reply:
These are properties of the ROM bootloader, separating them would be just for cosmetic reasons.

Reply:
I wouldn't say cosmetic, because in the proposed way FLASH_BT has now a dependency on another variable.
Being more explicit as @acassis suggested makes it easier to understand what is being passed to `esptool.py`.
Also, if the bootloader and partition table offsets are isolated, we can configure them with Kconfig later.

Reply:
I don't think we will ever change these configurations until we remove them all together. 
Anyway, here goes: done.